[2022-06-05 00:39:21,242] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T01:31:06.051317+00:00 [queued]>
[2022-06-05 00:39:21,273] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T01:31:06.051317+00:00 [queued]>
[2022-06-05 00:39:21,273] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 00:39:21,273] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 00:39:21,273] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 00:39:21,321] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): populate_table> on 2022-06-05 01:31:06.051317+00:00
[2022-06-05 00:39:21,340] {standard_task_runner.py:52} INFO - Started process 25908 to run task
[2022-06-05 00:39:21,346] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'populate_table', 'scheduled__2022-06-05T01:31:06.051317+00:00', '--job-id', '672', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpeixfi95c', '--error-file', '/tmp/tmp0fk02irw']
[2022-06-05 00:39:21,346] {standard_task_runner.py:80} INFO - Job 672: Subtask populate_table
[2022-06-05 00:39:21,474] {task_command.py:370} INFO - Running <TaskInstance: census.populate_table scheduled__2022-06-05T01:31:06.051317+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 00:39:21,613] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=populate_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T01:31:06.051317+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T01:31:06.051317+00:00
[2022-06-05 00:39:21,620] {logging_mixin.py:115} INFO - <_io.TextIOWrapper name='teste_uiliam.csv' mode='r' encoding='UTF-8'>
[2022-06-05 00:39:21,865] {python.py:173} INFO - Done. Returned value was: None
[2022-06-05 00:39:21,981] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=populate_table, execution_date=20220605T013106, start_date=20220605T033921, end_date=20220605T033921
[2022-06-05 00:39:22,042] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 00:39:22,097] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
